### ISSUE TITLE
Training file setup issues

### ISSUE THREAD

**Issue by GoPikachue** [icandle/MAN#7]
Hi, I'd like to start by asking some questions about the training file settings.
    First, I would like to ask if there is a problem with the number of iterations inside the scheduler settings, which is 1600K instead of 160K.
    Also, I found that you use MultiStepLR in the loss function setting instead of CosineAnnealingRestartLR as the paper says, which setting should I use better?
    Looking forward to your reply!!!
![UULXA4U`HT$ SL69SXDYQY8](https://user-images.githubusercontent.com/57658728/206888568-d7f7c90c-7837-4caf-9cd4-8d0376fd7632.png)

**Comment by icandle** [comment]
It is a mistake and should be 160k. In terms of schedule, both CosineAnnealingRestartLR and MultiStepLR would work. In our setting, we adopt the former while the latter is widely used in some other frameworks like HAT. We will correct the iter number and thanks for your comment.

**Comment by GoPikachue** [comment]
ok, thanks for your reply!
